src/webapp/components/collapsible-section/Section.tsx

```diff
@@ -13,53 +13,62 @@ export interface SectionProps {
     elevation?: number;
     iconPos?: iconPosition;
     classProps?: {
-        section?: string;
-        header?: string;
-        content?: string;
+        sectionPaper?: string;
+        sectionHeader?: string;
+        sectionContent?: string;
     };
     arrowStyle?: arrowStyle;
 }
 
-export const Section = ({
-    children,
-    title,
-    collapsible,
-    isOpen = true,
-    setOpen = () => {},
-    elevation = 1,
-    iconPos = "right",
-    classProps = {},
-    arrowStyle = "upDown",
-}: SectionProps) => {
+export const Section = (props: SectionProps) => {
+    const {
+        children,
+        title,
+        collapsible,
+        isOpen = true,
+        setOpen = () => {},
+        elevation = 1,
+        iconPos = "right",
+        classProps = {},
+        arrowStyle = "up_down",
+    } = props;
     const classes = useStyles();
     const leftIcon = iconPos === "left" ? classes.leftIcon : null;
     const toggle = () => setOpen(!isOpen);
 
+    const paperClasses = useMemo(() => ( _.compact([classes.paper, classProps.sectionPaper]).join(" ")),
+        [classes.paper, classProps.sectionPaper]);
+    const headerClasses = useMemo(() => ( _.compact([classes.header, leftIcon, classProps.sectionHeader, (isOpen ? null : classes.noBorder)]).join(" ")),
+        [classes.header, leftIcon, classProps.sectionHeader, isOpen, classes.noBorder]);
+    const contentClasses = useMemo(() => (_.compact([classProps.sectionContent, (isOpen ? null : classes.hidden)]).join(" ")),
+        [classProps.sectionContent, isOpen, classes.hidden]);
+
     return (
-        <Paper elevation={elevation} className={`${classes.paper} ${classProps.section}`}>
+        <Paper elevation={elevation} className={paperClasses}>
             <div
-                className={`${classes.header} ${leftIcon} ${classProps.header} ${isOpen ? "" : classes.noBorder}`}
+                className={headerClasses}
                 onClick={toggle}
             >
                 {collapsible && leftIcon && <CollapsibleToggle isOpen={isOpen} arrowStyle={arrowStyle} />}
                 {title}
                 {collapsible && !leftIcon && <CollapsibleToggle isOpen={isOpen} arrowStyle={arrowStyle} />}
             </div>
-            <div className={`${classProps.content}`} style={{ display: isOpen ? "" : "none" }}>
+            <div className={contentClasses}>
                 {children}
             </div>
         </Paper>
     );
 };
 
-export interface CollapsibleToggleProps {
+interface CollapsibleToggleProps {
```

Comment:
No need to change, just for future code, we now tend to use always "type" and "interface" only to define interface repositories and thinks like that, a type that someone does `implements xyz`.